hack/releasing/milestone_pull/README.md

```
@@ -0,0 +1,170 @@
+
+```bash
+git -C ../../kubernetes/test-infra diff --stat HEAD~1
+```
+
+Expect `config/prow/plugins.yaml | 3 ++-`. Anything else is a bug, not a surprise to accept.
+
+### Automatically, via the ChatOps command
+
+Commenting on the release issue runs the same script:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)